source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,162 @@
-using both HTML5 and [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) is to put the landmark on the sectioning
-element itself because this avoids the information being duplicated by
-screen readers that support both technologies.
+When using HTML5 elements such as `<main>`, ensure that you include a
+[shim that allows them to be styled](https://github.com/aFarkas/html5shiv) in
+legacy browsers. Include the script inside a `<!--[if lt IE 9]><![endif]-->`
+conditional comment so that modern browsers do not download them.
 
-Example:
+Your page should work with just HTML, before adding any CSS or JavaScript. Read
+[building a resilient frontend using progressive enhancement](https://www.gov.uk/service-manual/technology/using-progressive-enhancement).
+
+## Document structure
 
-    <header role="banner"></header>
+Use HTML5 to structure your page semantically. Use ARIA roles to help
+older assistive technologies map the sections correctly.
 
-The following roles map directly to HTML5 sectioning elements:
+Léonie Watson explains [why using ARIA landmark roles in this way delivers the
+best experience for screen reader users](http://tink.uk/screen-readers-aria-roles-html5-support/).
 
-* `role="main"` maps to the `<main>` element
-* `role="complementary"` maps to the `<aside>` element
-* `role="contentinfo"` maps to the `<footer>` element
-* `role="navigation"` maps to the `<nav>` element
+This document doesn't prescribe whether to use single or double quotes in attributes,
+whether to omit values from attributes that don't need them or whether to add
+trailing slashes to void elements. However, the approach you adopt should be
+consistent throughout your code.
 
-[This article from Léonie Watson](http://tink.uk/screen-readers-aria-roles-html5-support/) explains why using ARIA landmark roles in this way delivers the best experience for screen reader users.
+### Header
 
-For a more comprehensive guide to the mapping of roles to HTML5 sectioning elements read this [guide](http://blog.paciellogroup.com/2013/02/using-wai-aria-landmarks-2013/) from the Paciello Group.
+`<header role="banner">` should be used to contain your home link, branding, search,
+and any global navigation you may have.
 
-The following landmark roles don't map to HTML5 elements, but are
-still good to use:
+### Main content
 
-* `role="search"`
-* `role="application"`
+`<main role="main">` should be used to identify the main content of your page.
 
-[Demo of how landmark roles help screen reader users](http://tink.co.uk/2011/07/how-do-aria-landmark-roles-help-screen-reader-users/)
+A "Skip to main content" link should be added to help screen reader and keyboard
+users skip past your general site navigation and get straight to the content.
+You can use the [GOV.UK skip link component from the design system](https://design-system.service.gov.uk/components/skip-link/)
+for this.
 
-## Main content
+### Sectioned content
```

Comment:
I wonder if this might confuse people because GOV.UK doesn't use any `<section>`s anywhere.
Not a blocker at all, especially as sections have been in the text before, but something worth discussing. If we tell people to use it, why don't we use it ourselves?